ydb/core/health_check/health_check_ut.cpp

```diff
@@ -1818,123 +1834,128 @@ Y_UNIT_TEST_SUITE(THealthCheckTest) {
         UNIT_ASSERT_VALUES_EQUAL(database_status.storage().pools()[0].id(), "static");
     }
 
-    void HiveSyncTest(bool syncPeriod) {
```

Comment:
removed from main in https://github.com/ydb-platform/ydb/pull/9221/files#diff-675b73abe14f4d7620595d2b5662d9f20998e0a13abcdabb0c61889bff7b3c43